### ISSUE TITLE
Subprocesses not working in Docker containers...

### ISSUE THREAD

**Issue by eguerin-ucsf** [crispy-computing-machine/phpcrawl#14]
It doesn't seem like the application ever completed when running more then one process.  
I'm not sure if the processes are running and the execution of MPMODE_PARENT_EXECUTES_USERCODE  doesn't seem to be working, as I don't see console log output for the main process.

When i override the initChildProcess() function and print_r the $this i can the crawler object for the child process.

But I am not sure the child process is doing anything or talking back the main process.

Basically if i run with ->go() the whole scan is done fairly quickly.  If I run with goMultiProcessed() it doesn't appear anything is happening and the process never finishes.

**Comment by eguerin-ucsf** [comment]
This seems to be related to PHP 8.0 issues, they must have changed something to break the MultiProcessed functionality.   Updated Docksal container to use 7.4 and now things are working well.

**Comment by phptek** [comment]
Can confirm (if confirmation were needed) that this package has problems with PHP8.1 on Docker (Alpine Linux). I'm actually using the `->go()` method and in 8.1 I'm observing exactly the same behaviour as described by @eguerin-ucsf. 

Rebuilding the container to run with 7.4 and the exact same code - golden.

UPDATE: In my case, using the sqlite backend wasn't working. `$UrlDescriptor` (Line 627) was always false, presumably because Sqlite wasn't being updated. I switched to the memory backend - golden
